api/views.py

```diff
@@ -228,6 +228,8 @@ def get_random_message():
 
 def send_backUp_message(phone_number: str):
     apod_data = get_random_message()
+    #convert to dict
+    apod_data = apod_data.__dict__
```

Comment:
Python has a built in function for this
`apod_data = dict(apod_data)`

Reply:
gave an error so, I use `vars()